test/unit/job_test.rb

```diff
@@ -1460,7 +1451,7 @@
       Sidekiq.redis do |conn|
         assert_equal conn.zcard(Sidekiq::Cron::Job.new(@args).send(:job_enqueued_key)), 1, "Should have one enqueued job"
       end
-      assert_equal Sidekiq::Queue.all.first.size, 1, "Sidekiq queue 1 job in queue"
+      assert_equal Sidekiq::Queue.all.first&.size, 1, "Sidekiq queue 1 job in queue"
```

Comment:
@satyakampandya Why we need this change?

Reply:
To address the `Method invocation 'size' may produce 'NoMethodError'` warning, this change implements safe navigation.

Reply:
@satyakampandya Is this related to this PR? Not sure from where this warning is raised, but if this is unrelated to this PR, I'd like to revert those changes for now.